src/cache.js

```diff
@@ -74,5 +74,17 @@ function key (config) {
   return cacheKey
 }
 
-export { read, write, key }
-export default { read, write, key }
+function invalidate (config = {}) {
+  if (isFunction(config.invalidate)) return config.invalidate
+
+  return async (cfg, req) => {
+    const uuid = cfg.key(req)
```

Comment:
Here you don't need to compute the `uuid` from the `req`. It has already been done [in the `request` method](https://github.com/RasCarlito/axios-cache-adapter/pull/54/files#diff-aab1e3f14ac4fa89a7ece88d240d7a6eL8) and added to the `config` object.

So when we want to remove the item we can do
```js
await cfg.store.removeItem(cfg.uuid)
```

Reply:
Fixed